Amplify/Categories/DataStore/DataStoreCategory+HubPayloadEventName.swift

```diff
@@ -23,4 +23,44 @@ public extension HubPayload.EventName.DataStore {
     /// Dispatched when DataStore receives a sync response from the remote API via the API category. The Hub Payload
     /// will be a `MutationEvent` instance that caused the conditional save failed.
     static let conditionalSaveFailed = "DataStore.conditionalSaveFailed"
+
+    /// Dispatched when network status has changed (active or not)
+    /// The HubPayload will be a boolean value `isActive` indicating the status of network
+    // TODO: networkStatusChanged to be implemented
```

Comment:
can you also remove the events not used? reason is this could get released and then would never be emitted (not a huge deal), adding it later allows PR review to comment and help update the description, and there's no real reason now that you've narrowed down the scope of the PR